backend/src/error.rs

```diff
@@ -31,7 +31,7 @@ pub enum ErrorKind {
     InvalidOnionAddress = 22,
     Pack = 23,
     ValidateS9pk = 24,
-    DiskCorrupted = 25,
+    DiskCorrupted = 25, // Remove
```

Comment:
:question: Why the remove now? Is the `deprecated` better than remove?

Reply:
it has never been used. and I was just copying the comment from below